### ISSUE TITLE
`use_vars = false` does not work

### ISSUE THREAD

**Issue by iamishalkin** [taskipy/taskipy#63]
Hi
taskipy is great! However, I faced a minor glitch.

I'd like to run docker and pass the full path via `pwd` command to --volume argument. For some reason, taskipy interpretes `$(pwd)` as string. To overcome this problem, I found, that one could use `${PWD}` instead. However, in this case I get `the task "run_docker_dev" in the pyproject.toml file is malformed. reason: 'PWD' variable expected in [tool.taskipy.variables]` even if I pass `use_vars = false`

My task looks like this:
```
[tool.taskipy.tasks]
run_docker_dev = { cmd = "docker run --rm --volume ${PWD}/logging:/proj/logging proj:latest /bin/bash", use_vars = false }
```

I have taskipy version 1.11.0 and poetry 1.5.1, Win 11 with git bash. Both commands (with `${PWD}` and `$(pwd)`) work in bash terminal if I just copy-paste them from pyproject.toml

To sum up, I have two questions:
1. How to make taskipy execute `$(...)`? E.g., `echo = { cmd = "echo $(pwd)"}` returns `$(pwd)`
2. How to disable variables use in tasks?

**Comment by eugenetriguba** [comment]
Hey @iamishalkin, thanks for raising this :slightly_smiling_face: I was able to reproduce it and will take a look into resolving this

**Comment by illBeRoy** [comment]
Fixed in [taskipy@1.12.0](https://pypi.org/project/taskipy/1.12.0/)
